code/modules/jobs/job_types/security.dm

```diff
@@ -322,3 +322,35 @@ var/list/sec_departments = list("engineering", "supply", "medical", "science")
 /obj/item/device/radio/headset/headset_sec/alt/department/sci
 	keyslot = new /obj/item/device/encryptionkey/headset_sec
 	keyslot2 = new /obj/item/device/encryptionkey/headset_sci
+
+/datum/job/elo
+	title = "Elite Security Officer"
+	department_head = list("Head of Security")
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 3
+	spawn_positions = 3
+	supervisors = "the head of security"
+	selection_color = "#ffeeee"
+	minimal_player_age = 50
+	outfit = /datum/outfit/job/elo
+	access =  list(access_security, access_sec_doors, access_brig, access_armory, access_court, access_weapons,
+			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_medical)
+	minimal_access = list(access_security, access_sec_doors, access_brig, access_armory, access_court, access_weapons,
+			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_medical)
+	
+/datum/outfit/job/elo
+	name = "Elite Security Officer"
+	belt = /obj/item/weapon/storage/belt/military/full
+	ears = /obj/item/device/radio/headset/headset_sec/alt
+	uniform = /obj/item/clothing/under/rank/security/navyblue
+	shoes = /obj/item/clothing/shoes/combat/swat
+	suit = /obj/item/clothing/suit/armor/laserproof
+	suit_store = /obj/item/weapon/gun/energy/pulse/carbine

```

Comment:
you crazy, giving traitors eswords, ebows and holoparasites!! balance traitors and you will have a case. either way this job only has 2 slots open and there are normally only 4 traitors